README.md

```diff
@@ -1,4 +1,56 @@
-# flux-k8s
-Project to manage Flux tasks needed to standardize kubernetes HPC scheduling interfaces
+# Fluence
 
-Code under `golang` directory is under Apache v2.0 license.
+![docs/images/fluence.png](docs/images/fluence.png)
+
+Fluence enables HPC-grade pod scheduling in Kubernetes via the [Kubernetes Scheduling Framework](https://kubernetes.io/docs/concepts/scheduling-eviction/scheduling-framework/). Fluence uses the directed-graph based [Fluxion scheduler](https://github.com/flux-framework/flux-sched) to map pods or [podgroups](https://github.com/kubernetes-sigs/scheduler-plugins/tree/master/pkg/coscheduling) to nodes. Fluence supports all the Fluxion scheduling algorithms (e.g., `hi`, `low`, `hinode`, etc.). Note that Fluence does not currently support use in conjunction with the kube-scheduler. Pods must all be scheduled by Fluence.
+
+🚧️ Under Construction! 🚧️
+
+## Getting started
+For instructions on how to start Fluence on a K8s cluster, see [examples](examples/). Documentation and instructions for reproducing our CANOPIE2022 paper (citation below) can be found in the [canopie22-artifacts branch](/../canopie22-artifacts/canopie22-artifacts).
+
+## Papers
```

Comment:
One reading I found useful as a new person to this was https://ipo.llnl.gov/sites/default/files/2022-02/Flux_RD100_Final.pdf

I know its not a research paper but I thought it was a pretty good high level understanding.

Reply:
What parts of that document did you find the most helpful? (What should I include from it in this repo?)

Reply:
To be clear, I can certainly provide a link in the PR, but it might be good to use/adapt some of the helpful prose from the R&D100 submission.